packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1026,16 +1027,23 @@ export const VIDEO_RESOLUTIONS = {
 };
 
 /**
- * Max frame sizes based on h264 configs
+ * Max frame sizes and bandwith constraints based on h264 configs
  * https://en.wikipedia.org/wiki/Advanced_Video_Coding
  */
-export const MAX_FRAMESIZES = {
-  [QUALITY_LEVELS.LOW]: 1620,
-  [QUALITY_LEVELS.MEDIUM]: 3600,
-  [QUALITY_LEVELS.HIGH]: 8192
+ export const REMOTE_VIDEO_CONSTRAINTS = {
+  MAX_FS: {
+    [QUALITY_LEVELS.LOW]: 1620,
+    [QUALITY_LEVELS.MEDIUM]: 3600,
+    [QUALITY_LEVELS.HIGH]: 8192
+  },
+  // Decoder's maximum processing speed in macroblocks per second
+  MAX_MBPS: {   
+    [QUALITY_LEVELS.LOW]: 27600,
```

Comment:
<img width="814" alt="Screenshot 2022-10-21 at 1 17 08 PM" src="https://user-images.githubusercontent.com/16598065/197141976-d0ebe639-3fce-48b8-b1bd-522e13603c58.png">

Reply:
@arun3528 @marcin-bazyl

Reply:
Updated the value to 48600